lib/tftp/src/tftp.erl

```diff
@@ -243,158 +161,79 @@ of them differs a little. The available `option()`s are as follows:
   option `{udp, [{fd, 22}]}` is automatically added. See `init:get_argument/`
   about command-line arguments and `gen_udp:open/2` about UDP options.
 
-- **`{port_policy, Policy}`** -
-  `Policy = random | Port | {range, MinPort, MaxPort}`
-
-  `Port = MinPort = MaxPort = int()`
+- **`{port_policy, random | inet:port_number() | {range, Min::inet:port_number(), Max::inet:port_nuber()}`**
 
   Policy for the selection of the temporary port that is used by the
   server/client during the file transfer. Default is `random`, which is the
   standardized policy. With this policy a randomized free port is used. A single
   port or a range of ports can be useful if the protocol passes through a
   firewall.
 
-- **`{udp, Options}`** - `Options = [Opt]`, see
-  [gen_udp:open/2](`gen_udp:open/1`).
+- **`{udp, Options::gen_udp:option}`**
 
-- **`{use_tsize, Bool}`** - `Bool = bool()`
+- **`{use_tsize, boolean()}`**
 
   Flag for automated use of option `tsize`. With this set to `true`, the
   [`write_file/3`](`write_file/3`) client determines the filesize and sends it
   to the server as the standardized `tsize` option. A
   [`read_file/3`](`read_file/3`) client acquires only a filesize from the server
   by sending a zero `tsize`.
 
-- **`{max_tsize, MaxTsize}`** - `MaxTsize = int() | infinity`
+- **`{max_tsize, MaxTsize::pos_integer() | infinity}`**
```

Comment:
Here and in many places below `= int()` has been replaced with `:: pos_integer()`.  Is `0` really invalid in all these places, or should some be `:: non_neg_integer()`?